test/Confluent.Kafka.IntegrationTests/Tests/Consumer_QuickRebalance.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+#pragma warning disable xUnit1026
+
+using System;
+using Xunit;
+
+
+namespace Confluent.Kafka.IntegrationTests
+{
+    public partial class Tests
+    {
+        private static IConsumer<Null, string> quickRebalanceCreateConsumer(
+            String bootstrapServers, String groupId)
+        {
+            return new ConsumerBuilder<Null, string>(
+                new ConsumerConfig
+                {
+                    BootstrapServers = bootstrapServers,
+                    GroupId = groupId,
+                    AutoOffsetReset = AutoOffsetReset.Earliest,
+                    EnableAutoCommit = true,
+                    EnableAutoOffsetStore = false,
+                    AutoCommitIntervalMs = 500,
+                    PartitionAssignmentStrategy = PartitionAssignmentStrategy.RoundRobin,
+                })
+                .SetPartitionsAssignedHandler((c, partitions) => { })
```

Comment:
note, in C#, assign is not expected (or allowed) here.

Reply:
Hi @mhowlett 
What do you mean by not using assign from this handler - isn't it supported to return the exact topic partitions from this handler and then these will be used by the consumer?